core/sdk-core/src/main/java/software/amazon/awssdk/core/client/builder/SdkDefaultClientBuilder.java

```diff
@@ -133,12 +136,12 @@ public abstract class SdkDefaultClientBuilder<B extends SdkClientBuilder<B, C>,
     protected final SdkClientConfiguration.Builder clientConfiguration = SdkClientConfiguration.builder();
 
     protected final AttributeMap.Builder clientContextParams = AttributeMap.builder();
-
+    protected ClientOverrideConfiguration overrideConfig;
```

Comment:
The codegen clients builders need access to the client override configuration to properly configure the `RetryStrategy`.